src/components/BaseCard.vue

```diff
@@ -1,37 +1,37 @@
 <template>
   <div class="flex flex-col rounded-lg shadow-md overflow-hidden">
     <div class="flex-shrink-0">
-      <img class="h-48 w-full object-cover" :src="post.attributes.src || 'https://i.stack.imgur.com/y9DpT.jpg'">
+      <img class="h-48 w-full object-cover" :src="get(post, 'attributes.src', 'https://i.stack.imgur.com/y9DpT.jpg')">
     </div>
     <div class="flex-1 bg-white p-6 flex flex-col justify-between">
       <div class="flex-1">
         <p class="text-sm font-medium text-indigo-600">
           <span
-              v-for="(tag, index) in post.attributes.tags || []"
+              v-for="(tag, index) in get(post, 'attributes.tags', [])"
```

Comment:
We do not use the get method here, we try to keep the code clean and readable.
It is better to check the length of the tutor array this mean that on the p tag we will have a v-if directive like this:
`<p v-if="attributes.tags" class="text-sm font-medium text-indigo-600">`
Or we can use a computed method  ' tagsList' that returns this:
`return get(this.post, 'attributes.tags', [])`